packages/block-library/src/table/edit.js

```diff
@@ -377,15 +377,24 @@ export class TableEdit extends Component {
 							};
 
 							const cellClasses = classnames( { 'is-selected': isSelected } );
+							const richTextClassName = 'wp-block-table__cell-content';
```

Comment:
Could this just be `rich-text`?

Reply:
It could be, it's an existing class name though, not something new, so I don't see a big reason to change it.